templates/default/datadog.conf.erb

```diff
@@ -79,10 +79,10 @@ statsd_forward_port: <%= node['datadog']['statsd_forward_port'] %>
 
 log_level: <%= node['datadog']['log_level'] %>
 
-# collector_log_file: /var/log/datadog/collector.log
-# forwarder_log_file: /var/log/datadog/forwarder.log
-# dogstatsd_log_file: /var/log/datadog/dogstatsd.log
-# pup_log_file:       /var/log/datadog/pup.log
+collector_log_file: <%= node['datadog']['log_file_directory'] %>/collector.log
+forwarder_log_file: <%= node['datadog']['log_file_directory'] %>/forwarder.log
+dogstatsd_log_file: <%= node['datadog']['log_file_directory'] %>/dogstatsd.log
+pup_log_file:       <%= node['datadog']['log_file_directory'] %>/pup.log
```

Comment:
Pup was removed in https://github.com/DataDog/dd-agent/pull/1140

Might as well match the current log files: https://github.com/DataDog/dd-agent/blob/eab4b5f40af7002e1740b351d86eae20cf66ae1b/config.py#L938-L942